apartments/tests.py

```diff
@@ -3,43 +3,46 @@
 import pytest
 
 
+@pytest.fixture
```

Comment:
All of those fixtures exists also in `contacts/tests.py`, check if there is a way to use the same fixtures without creating duplicates.
I can see that it's not originated in this PR, so I'll approve it regardless, but create an issue and another PR to fix it (also check if it repeats in other places)

Reply:
Ok, I will check it and fix it in another PR.